.travis.yml

```diff
@@ -15,20 +15,25 @@ matrix:
   fast_finish: true
   include:
     - php: 7.3
-      env: RUN_TEST=true  ES_VERSION=73
+      env: RUN_TEST=true ES_VERSION=73
     - php: 7.4
-      env: RUN_TEST=true  ES_VERSION=73
+      env: RUN_TEST=true ES_VERSION=73
+    - php: nightly
+      env: RUN_TEST=true ES_VERSION=73 COMPOSER_FLAGS="--ignore-platform-reqs"
     # Testing the library with different ES versions
-    - php: 7.2
+    - php: 7.3
```

Comment:
Any chance you could push the changes for PHP 7.3 into a separate PR sow e can get it in quickly? Then this PR gets smaller and we know potential failures are only related to these changes.